### PR TITLE
Activitylog Service

### PR DESCRIPTION
**Pull request by kobergj**
Adds the activitylog service. As of now it only stores activities. Retrieving them comes with https://github.com/owncloud/ocis/issues/9196

For more info see also the ADR: https://owncloud.dev/ocis/adr/0028-activities-service/

Fixes https://github.com/owncloud/ocis/issues/9195

### CODE REVIEW

**Comment by kobergj** [comment]
- [ ] Provide a README.md for that service in the root folder of that service.
  - Use CamelCase for section headers.
- [ ] For images and example files used in README.md:
  - Create a folder named `md-sources` on the same level where README.md is located. Put all the images and example files referenced by README.md into this folder.
  - Use absolute references like `https://raw.githubusercontent.com/owncloud/ocis/master/services/<service-name>/md-sources/file` to make the content accessible for both README.md and owncloud.dev
    bad ``  
    good ``
- [ ] If new CLI command are introduced, that command must be described in the README.md.
- [ ] If new global envvars are introduced, the name must start with `OCIS_`.
- [ ] Add the service to the makefile in the ocis repo root.
- [ ] Make the service startable for binary and individual startup:
  - For single binary add service to `ocis/pkg/runtime`
  - For individual startup add service to `ocis/pkg/commands`
  - Add the service config to `ocis-pkg/config/defaultconfig.go`
- [ ] If the service is using service accounts, add it to `ocis/pkg/init/init.go`
- [ ] Add the service to `.drone.star` to enable CI.
- [ ] Inform doc team in an _early stage_ to review the readme AND the environment variables created.
  - The description must reflect the behaviour AND usually has a positive code quality impact.
- [ ] Create proper description strings for envvars - see other services for examples, especially when it comes to multiple values. This must include:
  - base description, set of available values, description of each value.
- [ ] When suggestable commits are created for text changes and you agree, collect them to a batch and commit them. Do not forget to rebase locally to avoid overwriting the changes made.
- [ ] If new envvars are introduced which serve the same purpose but in multiple services, an additional envvar must be added at the beginning of the list starting with `OCIS_` (global envvar).
- [ ] Ensure that a service has a debug port
- [ ] If the new service introduces a new port:
  - The port must be added to [port-ranges.md](https://github.com/owncloud/ocis/blob/master/docs/services/general-info/port-ranges.md) and to the README.md file.
- [ ] Make sure to have a function `FullDefaultConfig()` in `pkg/config/defaults/defaultconfig.go` of your service. It is needed to create the documentation.
- [ ] Add metrics to the code to enable monitoring. See the proxy service for implementation details.
  - Plus add a monitoring documentation in the README.md file

**Comment by mmattel** [comment]
Hooking myself in as this is also admin docs relevant.

**Review by 2403905 — APPROVED** [review]
Looks good for me.

File: services/activitylog/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Activitylog Service
+
+The `activitylog` service is responsible for storing events (activities) per resource.
+
+## The Log Service Ecosystem
+
+Log services like the `activitylog`, `userlog`, `clientlog` and `sse` are responsible for composing notifications for a certain audience.
```
Comment (dragonchaser) [inline services/activitylog/README.md:7]:
```suggestion
Log services like the `activitylog`, `userlog`, `clientlog` and `sse` are responsible for composing notifications for a specific audience.
```
Sounds better IMHO

**Review by dragonchaser — CHANGES_REQUESTED** [review]
Nit-picking on the wording.
and doesn't it need a change to the init command for the service account?

**Review by dragonchaser — APPROVED** [review]

**Pull request merged**